cep-XXXX.md

```diff
@@ -0,0 +1,216 @@
+# CEP XXXX - Names in conda packages and channels
+
+<table>
+<tr><td> Title </td><td> CEP XXXX - Names in conda packages and channels </td>
+<tr><td> Status </td><td> Draft </td></tr>
+<tr><td> Author(s) </td><td>
+  Jaime Rodríguez-Guerra &lt;jaime.rogue@gmail.com&gt; <br />
+  Matthew R. Becker &lt;becker.mr@gmail.com&gt; <br />
+  Cheng H. Lee &lt;clee@anaconda.com&gt;
+</td></tr>
+<tr><td> Created </td><td> Mar 11, 2025</td></tr>
+<tr><td> Updated </td><td> Apr 1, 2025</td></tr>
+<tr><td> Discussion </td><td> https://github.com/conda/ceps/pull/116 </td></tr>
+<tr><td> Implementation </td><td> N/A </td></tr>
+</table>
+
+## Abstract
+
+This CEP aims to standardize names and other strings used to identify packages, artifacts and
+channels in the conda ecosystem.
+
+## Specification
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT",
+"RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as
+described in [RFC2119][RFC2119] when, and only when, they appear in all capitals, as shown here.
+
+More specifically, violations of a MUST or MUST NOT rule MUST result in an error. Violations of the
+rules specified by any of the other all-capital terms MAY result in a warning, at discretion of the
+implementation.
+
+### Identifying package artifacts
+
+The conda ecosystem distinguishes between two types of packages:
+
+- Distributable package names: represented by a concrete, downloadable, extractable conda artifact.
+- Virtual package names: not backed by any physical artifact. They only exist on the client side.
+
+#### Package names
+
+A distributable package name MUST only consist of lowercase ASCII letters, numbers, hyphens, periods
+and underscores. It MUST start with a letter, a number, or a single underscore. It MUST NOT include
+two consecutive separators (hyphen, period, underscore).
+
+Virtual package names MUST only consist of lowercase ASCII letters, numbers, hyphens, periods and
+underscores. They MUST NOT use two consecutive separators, with one exception: they MUST start with
+two underscores.
+
+This means that distributable package names MUST match the following case-insensitive
+regex: `^(([a-z0-9])|([a-z0-9_](?!_)))[._-]?([a-z0-9]+(\.|-|_|$))*$`.
```

Comment:
Nit: let's move this before `Virtual packages` (line no. 45) for readability too

```suggestion
Distributable package names MUST match the following case-insensitive
regex: `^(([a-z0-9])|([a-z0-9_](?!_)))[._-]?([a-z0-9]+(\.|-|_|$))*$`.
```